client/sections.js

```diff
@@ -575,9 +575,16 @@ const sections = [
 		group: 'jetpack-cloud',
 		enableLoggedOut: true,
 	},
+	{
+		name: 'jetpack-cloud-manage-pricing',
+		paths: [ '/manage/pricing' ],
+		module: 'calypso/jetpack-cloud/sections/manage/pricing',
+		group: 'jetpack-cloud',
+		enableLoggedOut: true,
+	},
 	{
 		name: 'jetpack-cloud-pricing',
-		paths: [ '/pricing', '/[^\\/|^manage/]+/pricing', '/plans', '/[^\\/]+/plans' ],
+		paths: [ '/pricing', '/[^\\/]+/pricing', '/plans', '/[^\\/]+/plans' ],
```

Comment:
The downside to this fix is that it relies on entry order. I'd prefer something a bit more flexible, e.g.:

```suggestion
		paths: [ '/pricing', '/(?!manage)[^\\/]+/pricing', '/plans', '/[^\\/]+/plans' ],
```

Reply:
Yup, will look into this in a follow-up.
The current suggestion should work without introducing new regex issues, etc, so going "the safe route" for now :-)

Reply:
I think I would expect that such rules depend on entry order 🤔 But I agree it should be documented not coincidental

Reply:
> I would expect that such rules depend on entry order

I do prefer order to have semantic meaning in this case. However, given that we've got an array of paths, it's easy to envision a scenario like this:

* `paths: [ '/manage/pricing', '/[^\\/]+/plans', ],`
* `paths: [ '/[^\\/]+/pricing', '/manage/plans' ],`

In the above contrived example, `/manage/pricing` and `/manage/plans` would go with the first item. And that's with an easy example where the strings are easy enough to parse, but were the regex a bit more complex, keeping track of order is even more of a headache.

Reply:
Assuming languages are always two-char, this is a cleaner read:

`/[a-zA-Z]{2}/pricing`

Reply:
Noting I've added an issue here as the path change did result in styling changes on the page, so we do still need a follow-up: https://github.com/Automattic/jetpack-manage/issues/281